wafer/pages/models.py

```diff
@@ -6,11 +6,12 @@
 from django.core.exceptions import ValidationError, NON_FIELD_ERRORS
 from django.conf import settings
 from django.db import models
+from django.core.cache import caches
 from django.db.models.signals import post_save
 from django.utils.encoding import python_2_unicode_compatible
 
 
-from markitup.fields import MarkupField
+from markitup.fields import MarkupField, render_func
```

Comment:
django-markitup creates render_func with all the settings (see https://github.com/zsiciarz/django-markitup/blob/master/markitup/fields.py), and then uses it for the default rendering throughout.

We're copying the logic from MarkupField's pre_save method here, so we should be fine.